README.md

```diff
@@ -4,7 +4,7 @@
 [![GitHub forks](https://img.shields.io/github/forks/openjmu/OpenJMU)](https://github.com/openjmu/OpenJMU/network)
 [![GitHub issues](https://img.shields.io/github/issues/openjmu/OpenJMU)](https://github.com/openjmu/OpenJMU/issues)
 
-![](illustration.png)
+![illustration.png](https://i.loli.net/2020/02/15/BIwubHMLk3DRqfj.png)
```

Comment:
If the url will be converted, there's no need to use another place to store the image.

Reply:
But the actual use of the graph bed will speed up the loading of the README.